.github/workflows/danger-pr-labels.yml

```diff
@@ -10,9 +10,32 @@ jobs:
   check:
     runs-on: ubuntu-latest
     steps:
-      - name: Validate PR Labels
-        uses: danger/danger-js@9.1.8
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node_version: 10.x
+
+      - name: Install Yarn
+        run: npm install -g yarn
+
+      - name: Cache Node Modules
+        id: cache-node-modules
+        uses: actions/cache@v1
         with:
-          args: "--dangerfile Automattic/peril-settings/org/pr/label.ts"
+          path: node_modules
+          key: ${{ runner.os }}-node_modules-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: ${{ runner.os }}-node_modules
+
+      - name: Yarn Install
+        if: steps.cache-node-modules.outputs.cache-hit != 'true'
+        # frozen-lockfile will make the build fail if the lockfile is not there
+        run: yarn install --frozen-lockfile
```

Comment:
Only the output of `yarn install` is cached. I thought about making the `npm` command to install Yarn be a local one, to cache that output as well, but GitHub has already optimized `npm` runs and put in place their own caching for package used as much as Yarn.

<img width="1251" alt="Screen Shot 2020-02-17 at 11 03 38" src="https://user-images.githubusercontent.com/1218433/74615482-5b1ef380-5175-11ea-9db5-b9c6ad1b85a4.png">

Reply:
Given the fact that this is 1 second of the CI time, this seems like a sensible decision!